README.md

```diff
@@ -17,7 +17,9 @@
 
 ## Usage
 
-> Some usage instructions
+From within the root directory:
+
+> npm start
 
 ## Requirements
 
```

Comment:
@VALE46GP What version of node are you supposed to use? What database and what version of that database are you supposed to use?